intelmq/tests/bots/collectors/rsync/test_collector.py

```diff
@@ -17,7 +17,7 @@
           "raw": utils.base64_encode(EXAMPLE_FILE),
           }
 
-
+@unittest.skipIf(not os.path.exists("/opt/rsync"), "RSync is not installed")
```

Comment:
On Linux commands are not usually installed on `/opt`, but in `/bin` or `/usr/bin`

Reply:
What about using the return value of `type which`? Should be rather universal